ui/page/components/proposal_list.go

```diff
@@ -121,37 +120,57 @@ func layoutAuthorAndDate(gtx C, l *load.Load, item *ProposalItem) D {
 							return D{}
 						}),
 						layout.Rigid(timeAgoLabel.Layout),
-						layout.Rigid(func(gtx C) D {
-							if item.Proposal.Category == libwallet.ProposalCategoryPre {
-								return layout.Inset{Left: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
-									rect := image.Rectangle{
-										Min: gtx.Constraints.Min,
-										Max: gtx.Constraints.Max,
-									}
-									rect.Max.Y = 20
-									layoutInfoTooltip(gtx, rect, *item)
-
-									infoIcon := cryptomaterial.NewIcon(l.Theme.Icons.ActionInfo)
-									infoIcon.Color = l.Theme.Color.GrayText2
-									return infoIcon.Layout(gtx, values.MarginPadding20)
-								})
-							}
-							return D{}
-						}),
 					)
 				}),
 			)
 		}),
 	)
 }
 
-func layoutTitle(gtx C, l *load.Load, proposal libwallet.Proposal) D {
-	lbl := l.Theme.H6(proposal.Name)
-	lbl.Font.Weight = font.SemiBold
-	return layout.Inset{Top: values.MarginPadding4}.Layout(gtx, lbl.Layout)
+func layoutAuthor(gtx C, l *load.Load, item *ProposalItem) D {
+	proposal := item.Proposal
+	grayCol := l.Theme.Color.GrayText2
+
+	nameLabel := l.Theme.Body2(proposal.Username)
+	nameLabel.Color = grayCol
+
+	dotLabel := l.Theme.H4(" . ")
+	dotLabel.Color = grayCol
+
+	stateLabel := getStateLabel(l, proposal)
+
+	timeAgoLabel := l.Theme.Body2(TimeAgo(proposal.Timestamp))
+	timeAgoLabel.Color = grayCol
+
+	versionLabel := l.Theme.Body2(values.String(values.StrVersion) + " " + proposal.Version)
+	versionLabel.Color = grayCol
+
+	return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layout.Flex{}.Layout(gtx,
+				layout.Rigid(nameLabel.Layout),
+				layout.Rigid(func(gtx C) D {
+					return layout.Inset{Top: values.MarginPaddingMinus22}.Layout(gtx, dotLabel.Layout)
+				}),
+				layout.Rigid(versionLabel.Layout),
+			)
+		}),
+		layout.Rigid(func(gtx C) D {
+			if proposal.Category == libwallet.ProposalCategoryPre {
+				return layout.Flex{}.Layout(gtx,
+					layout.Rigid(stateLabel.Layout),
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Top: values.MarginPaddingMinus22}.Layout(gtx, dotLabel.Layout)
+					}),
+					layout.Rigid(timeAgoLabel.Layout),
+				)
+			}
+			return D{}
```

Comment:
The smaller block of code should be in the if statement i.e `if proposal.Category != libwallet.ProposalCategoryPre { return D{} }  .....`